patent.py

```diff
@@ -0,0 +1,40 @@
+import argparse
+import requests
+import json
+
+
+def patent(**kwargs):
+    patent_url = 'https://api.nasa.gov/patents/content'
+    get_params = {}
```

Comment:
I am not understanding this and the next two lines. You essentially create a dict `get_params` then add all of the key/value pairs from `kwargs`, making it identical. Then you never use `get_params`. Was there a reason for this?